__tests__/app.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe('ENDPOINT TESTS', () => {
+    describe('GET /api/topics', () => {
+        test('GET 200 - should return all an array with all topics from db', () => {
+            return request(app)
+                .get('/api/topics')
+                .expect(200)
+                .then((res) => {
+                    expect(res.body.topics).toBeInstanceOf(Array)
+                    expect(res.body.topics.length).toBe(3)
```

Comment:
specifying a number here has the potential to make this test brittle if new catergories were ever added to the test data, I'd recommend testing for greater than 0.

Reply:
Fixed as per your recommendation